src/screens/ConfirmationContainer/ConfirmationScreen/index.js

```diff
@@ -179,7 +179,7 @@ class ConfirmationScreen extends NavigatorComponent<Props, State> {
                 <TextInput
                   style={[styles.textInputConfirmation, styles.bodyBoldBlack]}
                   onChangeText={gasLimit => this.setState({ gasLimit })}
-                  value={this.state.gasLimit}
+                  value={this.state.gasLimit.toString()}
```

Comment:
If `gasLimit` is a string, is that required to apply `toString` on it?

Reply:
This is just to be assure consistency because `gasLimit` is part of the Transaction object and can be sent to `SendMoney` even from a DApp without type check, and that could lead to a render error on the TextInput component

Reply:
Idea is that type annotation should always match what we expect to be passed there. If we imagine that it can be also not a string, then we need to annotate it somehow another way.

Reply:
Ok, solved this into the `CustomSigner.js` file